### ISSUE TITLE
Why does my interface need 500ms after using this adapter

### ISSUE THREAD

**Issue by Peyton-lee** [senecajs/seneca-web-adapter-express#20]
```js
const express = require('express');
const Seneca = require('seneca');
const SenecaWeb = require('seneca-web');
const compression = require('compression');
const cookieParser = require('cookie-parser');
const bodyParser = require('body-parser');
const routes = require('./routes');
const adapter = require('seneca-web-adapter-express');
const IS_DEBUG = process.env.NODE_ENV !== 'production';
const ServiceConfs = IS_DEBUG ? require('../configs/service.dev.json') : require('../configs/service.prod.json');

const seneca = Seneca()
  .test('print')
  .use(SenecaWeb, {
    context: (() => {
      const app = express();
      app.use(bodyParser.json());
      app.use(compression());
      app.use(bodyParser.urlencoded({ extended: false }))
      app.use(cookieParser());
      return app;
    })(),
    adapter,
    routes,
  })
  .use("entity", {
    mem_store: false
  })
  .client(ServiceConfs['seneca-srv1'])
  .client(ServiceConfs['order-srv'])
  .ready(async() => {
    const app = seneca.export('web/context')();
    const port = process.env.PORT;
    app.listen(port, () => {
      console.log(`Api Server listen on: ${port}`);
    });
  });

```

![image](https://user-images.githubusercontent.com/17973574/64684021-e86b6700-d4b6-11e9-929b-6bfa66930015.png)

**Comment by Peyton-lee** [comment]
database search consume 40ms

**Comment by Peyton-lee** [comment]
```js
    const payload = {
      request$: request,
      response$: reply,
      args: {
        body: body,
        route: route,
        params: request.params,
        query: request.query,
        user: request.user || null
      }
    }

    // Call the seneca action specified in the config
    seneca.act(route.pattern, payload, (err, response) => {
      if (err) {
        return next(err)
      }

      // Redirect or reply depending on config.
      if (route.redirect) {
        return reply.redirect(route.redirect)
      }
      if (route.autoreply) {
        return reply.send(response)
      }
    })
  }
```

I found that it's because of this "payload" parameter is too large.
How should I fix it?

**Comment by tswaters** [comment]
Yea, so adding an option to the adapter and running for both, got a bit of nonsense result -- including it was actually faster with some one of the tests?  

Either way, it reports "Basically the same" in cases where test('print') is off -- and test('print') on is slow as you've noted...  anyway, you can now disable it if you want.  Right now it's disabled to be enabled -- I'll make it disabled by default next major, and remove the option major after that.

I've published this as [v1.2.0.](https://github.com/senecajs/seneca-web-adapter-express/releases/tag/v1.2.0)
